src/UWP/ArcGISRuntime.UWP.Viewer/Samples/GeometryEngine/Buffer/Buffer.xaml

```diff
@@ -1,23 +1,26 @@
 ﻿<UserControl
     x:Class="ArcGISRuntime.UWP.Samples.Buffer.Buffer"
-    x:Name="_this"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:ArcGISRuntime.UWP.Samples.Buffer"
     xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
     <Grid>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
-        </Grid.RowDefinitions>
-        <StackPanel Grid.Row="0" 
-                    Orientation="Horizontal" >
-            <TextBlock x:Name="BufferInstructionsLabel" 
-                       Text="Specify the buffer in miles and the tap on the map:"/>
-            <TextBox x:Name="BufferDistanceMilesTextBox" 
-                     Text="10"/>
-        </StackPanel>
-        <esriUI:MapView x:Name="MyMapView" 
-                        Grid.Row="1"/>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border BorderBrush="Black" BorderThickness="1" Background="White"
+                HorizontalAlignment="Right" VerticalAlignment="Top"
+                Margin="30" Padding="20" Width="375">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="*" />
+                </Grid.ColumnDefinitions>
+                <TextBlock Text="Specify the buffer in miles and the tap on the map:"
```

Comment:
This XAML produces something different from the screenshot above. The label does not fit into its column:

![2018-06-12_11-03-59_applicationframehost](https://user-images.githubusercontent.com/587809/41308188-16e1f732-6e30-11e8-911d-ccc02d3b58e9.png)